collector/collector.go

```diff
@@ -145,6 +146,27 @@ func (c *Collector) collectAllQueues(httpClient *http.Client, result *Result) er
 
 	if c.DebugHttp {
 		trace := &httptrace.ClientTrace{
+			GetConn: func(hostPort string) {
+				fmt.Printf("getting connection to %s: %v\n", hostPort, time.Now())
+			},
+			GotConn: func(info httptrace.GotConnInfo) {
+				fmt.Printf("got connection [reused?: %t, idle?: %t (time: %v)]: %v\n",
+					info.Reused, info.WasIdle, info.IdleTime, time.Now())
+			},
+			PutIdleConn: func(err error) {
+				if err == nil {
+					fmt.Printf("connection successfully put idle: %v\n", time.Now())
+				} else {
+					fmt.Printf("Failed to put connection idle with error - %v: %v\n", err, time.Now())
+				}
```

Comment:
`if err == nil` looks a bit odd (from a Go perspective). https://google.github.io/styleguide/go/decisions#indent-error-flow

```suggestion
				if err != nil { {
					fmt.Printf("Failed to put connection idle with error - %v: %v\n", err, time.Now())
					return
				}
				fmt.Printf("connection successfully put idle: %v\n", time.Now())
```